client/bin/multi-client.js

```diff
@@ -371,82 +380,82 @@ class NonInteractiveState {
 						}`,
 					);
 				}
-			}
 
-			if (job.workers === null) {
-				// No idle workers currently - the job is pushed to the back of the queue
-				await require('bluebird').delay(25000);
-				runQueue.unshift(job);
-			} else {
-				// Start the job on the assigned worker
-				state.attachPanel(job);
-				const child = fork(
-					path.join(__dirname, 'single-client'),
-					[
-						'-d',
-						job.deviceType,
-						'-i',
-						job.image,
-						'-c',
-						job.config instanceof Object
-							? JSON.stringify(job.config)
-							: job.config,
-						'-s',
-						job.suite,
-						'-u',
-						job.workers,
-					],
-					{
-						stdio: 'pipe',
-						env: {
-							CI: true,
+				if (job.workers === null) {
+					// No idle workers currently - the job is pushed to the back of the queue
+					await require('bluebird').delay(25000);
+					runQueue.unshift(job);
+				} else {
+					// Start the job on the assigned worker
+					state.attachPanel(job);
+					const child = fork(
+						path.join(__dirname, 'single-client'),
+						[
+							'-d',
+							job.deviceType,
+							'-i',
+							job.image,
+							'-c',
+							job.config instanceof Object
+								? JSON.stringify(job.config)
+								: job.config,
+							'-s',
+							job.suite,
+							'-u',
+							job.workers,
+						],
+						{
+							stdio: 'pipe',
+							env: {
+								CI: true,
+							},
```

Comment:
Can you move all these linting changes into a separate commit?

Reply:
Done, and probably it was a false negative